index.html

```diff
@@ -1,70 +1,218 @@
+        <p>A daily selection of privately personalized reads; 
+          no accounts or sign-ups required</p>
+        <ul class="keyword">
+          <li>html</li>
+          <li>css</li>
+          <li>javaScript</li>
+        </ul>
+        <div class="cta-button">See Project</div>
+      </div>
 
-  </div>
+      <div class="bg-card">
+        <img src="images/mps-1.png" alt="MPS project seakpeak">
+        <h1>Multi-Post 
+          <br>Stories</h1>
+        <ul class="categories">
+          <li class="canopy">CANOPY</li>
+          <li>•</li>
+          <li>Back End Dev</li>
+          <li>•</li>
+          <li>2015</li>
+        </ul>
+        <p>A daily selection of privately personalized reads; 
+          no accounts or sign-ups required</p>
+        <ul class="keyword">
+          <li>html</li>
+          <li>css</li>
+          <li>javaScript</li>
+        </ul>
+        <div class="cta-button">See Project</div>
+      </div>
 
-  <div class="footer">
-    <h3>LET'S CONNECT</h3>
-  <ul class="icon-bar">
-    <li><a href="https://twitter.com/87juda">
-      <img src="images/twitter.png" alt="Twitter">
-    </a></li>
-    <li><a href="https://www.linkedin.com/in/judateixeira/">
-      <img src="images/linkedin.png" alt="Linkedin">
-    </a></li>
-    <li><a href="https://web.facebook.com/judateixeira/">
-      <img src="images/facebook.png" alt="Facebook">
-    </a></li>
-    <li><a href="https://github.com/mrjuda">
-      <img src="images/github.png" alt="Github">
-    </a></li>
-    <li><a href="https://instagram.com/srjuda/">
-      <img src="images/instagram.png" alt="Facebook">
-    </a></li>
-  </ul>
-  </div>
+      <div class="bg-card">
+        <img src="images/tonic-2.png" alt="Tonic project seakpeak">
+        <h1>Tonic</h1>
+        <ul class="categories">
+          <li class="canopy">CANOPY</li>
+          <li>•</li>
+          <li>Back End Dev</li>
+          <li>•</li>
+          <li>2015</li>
+        </ul>
+        <p>A daily selection of privately personalized reads; 
+          no accounts or sign-ups required</p>
+        <ul class="keyword">
+          <li>html</li>
+          <li>css</li>
+          <li>javaScript</li>
+        </ul>
+        <div class="cta-button">See Project</div>
+      </div>
 
-</div>
+      <div class="bg-card">
+        <img src="images/mps-2.png" alt="Tonic project seakpeak">
+        <h1>Multi-Post 
+          <br>Stories</h1>
+        <ul class="categories">
+          <li class="canopy">CANOPY</li>
+          <li>•</li>
+          <li>Back End Dev</li>
+          <li>•</li>
+          <li>2015</li>
+        </ul>
+        <p>A daily selection of privately personalized reads; 
+          no accounts or sign-ups required</p>
+        <ul class="keyword">
+          <li>html</li>
+          <li>css</li>
+          <li>javaScript</li>
+        </ul>
+        <div class="cta-button">See Project</div>
+      </div>
+    </div> <!-- End of WORKS -->
 
+    <div class="about" id="bg-about">
```

Comment:
- [ ] As per project requirements, the container of the section is meant to be a `<section>` tag. Implementing this will improve the overall readability of the code while encouraging the use of semantic HTML tags. 😄
- [ ] In reference to the project requirements, 👇🏻 It would be nice if you aligned the project elements with `flexbox`.
<img width="260" alt="try" src="https://user-images.githubusercontent.com/4247740/144695867-75555510-d106-47fa-81b4-f3f554b9828f.PNG">